freqtrade/freqtradebot.py

```diff
@@ -639,7 +639,6 @@ def execute_entry(
                 entry_tag=enter_tag, side=trade_side):
             logger.info(f"User requested abortion of buying {pair}")
             return False
-        amount = self.exchange.amount_to_precision(pair, amount)
```

Comment:
We don't need this here as it's called in create_order, too (and there the call is correct, calculating also the contract sizes).

Reply:
@samgermain would you mind having a look at this again (not this line in particular)

we seem to have only one instance of `amount = self.amount_to_precision(pair, self._amount_to_contracts(pair, amount))` - all others omit the `amount_to_contracts` part.

is that correct (for example for dry-run)? and is it correct to not need this to place stoploss orders?